maskrcnn_benchmark/engine/inference.py

```diff
@@ -87,8 +87,11 @@ def prepare_for_coco_segmentation(predictions, dataset):
         image_height = dataset.coco.imgs[original_id]["height"]
         prediction = prediction.resize((image_width, image_height))
         masks = prediction.get_field("mask")
+        
         # t = time.time()
-        masks = masker(masks, prediction)
+        # Masker is necessary only if masks haven't been already resized.
+        if list(masks.shape[-2:]) != [image_width, image_height]:
```

Comment:
I think this is a bit fragile. Ideally, we would want the `Masker` to return a sequence of `Mask` objects, but this is not currently fully implemented, so we might want to let it for later.

Also, I think this should be `[image_height, image_width]`, as the shape of tensors is represented with `[N,C,H,W]` order generally.

Reply:
Yes, ideally we would like Masker to be able to handle multi images batch.